contrib/platform/src/com/sun/jna/platform/win32/Psapi.java

```diff
@@ -266,6 +266,46 @@
      */
     boolean GetPerformanceInfo(PERFORMANCE_INFORMATION pPerformanceInformation, int cb);
 
+    /**
+     * It is a good idea to use a large array, because it is hard to predict
```

Comment:
Please start this comment with the short blurb at the top of the API, "Retrieves the process identifier for each process object in the system."  After that, following with the remarks as you've done is good.

Reply:
I did same as for the other methods that were mapped (copied the text from MSDN), but sure. I can do that.

Reply:
There's two sections. The top of the page is actually the method description.  The section you copied is the remarks, which usually adds useful information about using the function (such as the fact that if lengths are equal you should call it again) which should also be added.